### PR TITLE
Improve output of `hab svc status` command

### PR DESCRIPTION
**Pull request by reset**
* Formatting changes to be more friendly for machines to read
* Add service group to output
* Add style to output

screen shot 2017-05-18 at 8 02 04 pm

Addresses #2334 & #2274

The line length would get pretty long if we included as much information requested in #2334. We may want to revisit this and either:

* Add another command which outputs full information about a service
* Modify this command so that if it receives an argument for a particular package we output more information

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/sup/src/main.rs [hunk 1]
```diff
@@ -358,53 +358,32 @@ fn sub_status(m: &ArgMatches) -> Result<()> {
...
+            if services.is_empty() {
+                println!("No services loaded.");
+                return Ok(());
             }
-            ProcessState::Restart => {
-                outputln!("The {} service entered the restarting state {} seconds ago.",
-                          status.pkg.ident,
-                          status.process.elapsed.num_seconds());
+            for status in services {
+                println!("{}", status);
```
Comment (reset) [inline components/sup/src/main.rs]: I did intentionall change these from `outputln!` to `println!`. The intent of `outputln!` was to let us know which module a log message was coming from and if it was from a service or from the supervisor itself. That information isn't very useful when we're just dealing with one-off commands like this

File: components/sup/src/main.rs [hunk 2]
```diff
@@ -358,53 +358,32 @@ fn sub_status(m: &ArgMatches) -> Result<()> {
...
-            }
-            ProcessState::Down => {
-                outputln!("The {} service is not currently running.", status.pkg.ident);
-            }
-            ProcessState::Start => {
-                outputln!("The {} service entered the starting state {} seconds ago.",
-                          status.pkg.ident,
-                          status.process.elapsed.num_seconds());
+        None => {
+            let services = Manager::status(cfg)?;
```
Comment (christophermaier) [inline components/sup/src/main.rs]: The equating of "services" with "status" reads confusingly... are these `ServiceStatus` structs?

**Comment by christophermaier** [comment]
Aside from a naming comment... 
26b3f32e-511b-4009-bfb2-a6877a9af493

**Comment by reset** [comment]
@christophermaier good catch on that var name, it's statuses which are a subset of a service itself. Thanks!

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/233872005).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/233872005).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**